### ISSUE TITLE
UI Press - create Book content type

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#4328]
This will replace UI Press Book content type currently used on their d7 site.

General layout - customer would like something similar to an amazon book page, with the book cover on the left, text about the book in the middle, and buying options on the right. Book cover image needs to be larger than it is on current site.

Fields & their current formats, with some possible changes:
- Title - text field, required
- Subtitle - text field
- Book cover - image upload.
- Author(s) - autocomplete from Person content type
- Editor(s) - autocomplete from Person content type
- Illustrator(s) - autocomplete from Person content type
- Photographer(s) - autocomplete from Person content type
- Translator(s) - autocomplete from Person content type
- Preface author(s) - autocomplete from Person content type
- Foreword author(s) - autocomplete from Person content type
- Introduction author(s) - autocomplete from Person content type
- Afterword author(s) - autocomplete from Person content type
- Essay by - autocomplete from Person content type
- ~As told to - autocomplete from Person content type~ checking if needed
- Awards - text area
- Publication date - should display month & year
- Pages - text field
- Art - text field
- Trim size - text field
- Season - taxonomy (new vocab)
- Edition - radio button selection, required. Options: 1st, 2nd, 3rd, 4th, 5th.
- Hardcover retail price - decimal field
- Hardcover sale price -  decimal field
- Hardcover sale code - text field
- Hardcover sale start date - mm/dd/yyyy
- Hardcover sale expiry date - mm/dd/yyyy
- Hardcover ISBN-13 - string
- Paperback retail price - decimal field
- Paperback sale price -  decimal field
- Paperback sale code - text field
- Paperback sale start date - mm/dd/yyyy
- Paperback sale expiry date - mm/dd/yyyy
- Paperback ISBN-13 - string
- eBook retail price - decimal field
- eBook sale price -  decimal field
- eBook sale code - text field
- eBook sale start date - mm/dd/yyyy
- eBook sale expiry date - mm/dd/yyyy
- eBook ISBN-13 - string
- Reviews/blurbs - text area
- Book description/summary - text area
- Contributors - text area
- Table of contents - text area
- Excerpt - text area 
- Series - term reference, unlimited. Add Series as a vocabulary.
- Subject(s) - term reference, unlimited. Add Subjects as a vocabulary.
- Related Reads - autocomplete from Book content type, would like option for 5 related reads
- File uploads - they would like a section to upload images from the book, shared example of site that had modal image gallery.

They would like a way to share the book info straight to social media, like the 'Share Title' section [halfway down this page](https://www.graywolfpress.org/books/my-name-will-grow-wide-tree). Did not commit, said we would investigate further.

Instead of listing every field for Hardcover, Paperback, and eBook, it would be better to have them as sections that can be expanded as needed. Also, sale prices should only display when a sale is active.

**Comment by briand44** [comment]
potentially use a paragraph bundle with book type for related fields (hardcover, paperback, etc)

**Comment by joewhitsitt** [comment]
moved back to planned under the line as we weren't sure anything had been pushed up for others to take on while Makur was out

**Comment by makurj** [comment]
Waiting on https://github.com/uiowa/uiowa/issues/4329 to create "autocomplete from Person content type" fields.

**Comment by makurj** [comment]
Need to figure out the layout of the page. Should that be a separate issue?

**Comment by briand44** [comment]
@makurj I think that is this issue. https://github.com/uiowa/uiowa/issues/4522

**Comment by makurj** [comment]
Thanks @dnkemper for all her continued and patient help on this!
